server/daemon/src/main.rs

```diff
@@ -771,15 +763,11 @@ async fn kanidm_main(config: Configuration, opt: KanidmdParser) -> ExitCode {
                                                     // systemd has a special reload handler for this.
                                                     #[cfg(target_os = "linux")]
                                                     {
-                                                    let _ = sd_notify::notify(true, &[sd_notify::NotifyState::Reloading]);
-                                                    // CRITICAL - if you do not send a monotonic usec message after a reloading
-                                                    // message, your service WILL BE KILLED.
                                 if let Ok(monotonic_usec) = sd_notify::NotifyState::monotonic_usec_now() {
-                                let _ =
-                                    sd_notify::notify(true, &[monotonic_usec]);
-                                } else {
+                                else {
```

Comment:
I think I accidentally a bracket here: `}`
```suggestion
                               } else {
```